autoscheduler/frontend/src/components/SchedulingPage/ConfigureCard/ConfigureCard.tsx

```diff
@@ -25,95 +16,29 @@ import { formatTime } from '../../../utils/timeUtil';
 const ConfigureCard: React.FC = () => {
   const [includeFull, setIncludeFull] = React.useState(false);
   const [loading, setLoading] = React.useState(false);
-  const [showSnackbar, setShowSnackbar] = React.useState(false);
-
-  const courseCards = useSelector<RootState, CourseCardArray>((state) => state.courseCards);
-  const term = useSelector<RootState, string>((state) => state.term);
-  const avsList = useSelector<RootState, Availability[]>((state) => state.availability);
-  const dispatch = useDispatch();
+  const [snackbarMessage, setSnackbarMessage] = React.useState('');
+  const dispatch = useThunkDispatch();
 
   // Holds a reference to the DOM element to check if the component is still mounted
   const isMounted = React.useRef(true);
   React.useEffect((): VoidFunction => (): void => {
     isMounted.current = false;
   }, []);
 
-  const checkIfEmpty = (schedules: Meeting[][]): Meeting[][] => {
-    if (isMounted.current && schedules.length === 0) setShowSnackbar(true);
-    return schedules;
-  };
-
   // closes the snackbar if the user presses the close icon, but not if they click away
   const handleSnackbarClose = (_event: any, reason: string): void => {
     if (reason === 'clickaway') return;
-    setShowSnackbar(false);
+    setSnackbarMessage('');
   };
 
   const fetchSchedules = React.useCallback(() => {
```

Comment:
Is it still worth using `useCallback` here? Since all of the logic got moved to the action it seems like this wouldn't increase performance any more